src/test/suite/format/samples/255-nested-one-command-code.in.ahk

```diff
@@ -0,0 +1,407 @@
+; [Issue #255](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/255)
+if {
```

Comment:
Can we split this into multiple tests? 400+ lines is huge, we should aim for less than 20 lines per sample file if possible.

Each file can have the `255` prefix still, no reason there can't be more than one test file for a bug :)

Reply:
Done.